cypress/e2e/SecuritySettings.cy.js

```diff
@@ -7,6 +7,11 @@ beforeEach(() => {
   cy.visit('/security-settings')
 })
 
+it('Dashboard has no detectable a11y violations on load', () => {
```

Comment:
While I like that you're adding this, please update the descriptions for each one to reflect the page the test is being ran on (in this case Security settings).

Reply:
Haha, right, Good catch